.orchestra/ci/install-system-dependencies.sh

```diff
@@ -95,3 +95,6 @@ if ! which git-lfs &> /dev/null; then
   curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
   apt-get -qq install --no-install-recommends --yes git-lfs
 fi
+
+yarn global add prettier
+
```

Comment:
* `yarn global add prettier`: where does this put things? This command will run as `root`.
* Drop trailing space

Reply:
Mmmmh, I'm not very happy to have someone who's not the package manager install stuff in `/usr`.
I see there's a package on `snap` but requiring users to use `snap` is a bit sad.
Can we run it as non-root (as we're doing with `install-python-dependencies.sh`) and install it in `~/.local/bin`?
Otherwise, we can make an orchestra component out of this and install it in `$ORCHESTRA_ROOT/node_modules`, which we already have.

Reply:
Great! `:)`